src/org/omegat/core/dictionaries/EBDict.java

```diff
@@ -0,0 +1,372 @@
+            output.append("<sub>");
+        }
+
+        /**
+         * end subscript
+         */
+        @Override
+        public void endSubscript() {
+            output.append("</sub>");
+        }
+
+        /**
+         * begin super script
+         */
+        @Override
+        public void beginSuperscript() {
+            output.append("<sup>");
+        }
+
+        /**
+         * end super script
+         */
+        @Override
+        public void endSuperscript() {
+            output.append("</sup>");
+        }
+
+        /**
+         * set indent of line head
+         * 
+         * @param len 
+         */
+        @Override
+        public void setIndent(int len) {
+            for (int i = 0 ; i<len; i++ ) {
+                output.append("&nbsp;");
+            }
+        }
+
+        /**
+         * insert new line.
+         */
+        @Override
+        public void newLine() {
+            output.append("<br>");
+            lineNum++;
+        }
+
+        /**
+         * set no break
+         */
+        @Override
+        public void beginNoNewLine() {
+            // FIXME: implement me.
+        }
+        @Override
+        public void endNoNewLine() {
+            // FIXME
+        }
+
+        /**
+         * insert em tag
+         */
+        @Override
+        public void beginEmphasis() {
+            output.append("<em>");
+        }
+        @Override
+        public void endEmphasis() {
+            output.append("</em>");
+        }
+
+        /**
+         * insert decoretion
+         *
+         * @param type decoration type
+         * #BOLD
+         * #ITALIC
+         */
+        @Override
+        public void beginDecoration(int type) {
+            this.decType = type;
+            switch (decType) {
+                case BOLD:
+                    output.append("<i>");
+                    break;
+                case ITALIC:
+                    output.append("<b>");
+                    break;
+                default:
+                    output.append("<u>");
+                    break;
+            }
+        }
+
+        @Override
+        public void endDecoration() {
+             switch (decType) {
+                case BOLD:
+                    output.append("</i>");
+                    break;
+                case ITALIC:
+                    output.append("</b>");
+                    break;
+                default:
+                    output.append("</u>");
+                    break;
+            }
+        }
+
+        /**
+         * convert Zenkaku alphabet to Hankaku
+         *
+         * convert (\uFF01 - \uFF5E) to (\u0021- \u007E)
+         *
+         * @param text
+         * @return String converted
+        */
+        public String convertZen2Han(String text) {
+            int i;
+            StringBuilder result = new StringBuilder(text.length());
+            for (i = 0; i < text.length(); i++ ) {
+                char c = text.charAt(i);
```

Comment:
EPWING standard use Shift_JIS for character encoding. It is why charAt() is safe here.